content/docs/setup/kubernetes/install/cni/index.md

```diff
@@ -45,30 +45,73 @@ replaces the functionality provided by the `istio-init` container.
     1.  Refer to the [Hosted Kubernetes Usage](#hosted-kubernetes-usage) section for any non-default
         settings required.
 
-1.  Add the `--set istio_cni.enabled=true` setting to the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/) to include the installation of the Istio CNI plugin. For example:
+1.  Update Helm's local package cache per the instructions in the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/).
+
+    1.  If installing via `helm template` fetch the `istio-cni` template in addition to the other Istio
+        templates fetched per the [Istio Helm installation procedure](/docs/setup/kubernetes/install/helm/#option-1-install-with-helm-via-helm-template)
+
+        {{< text bash >}}
+        $ helm fetch istio.io/istio-cni --untar --untardir $HOME/istio-fetch
+        {{< /text >}}
+
+1.  Create a namespace for the Istio control-plane components:
 
     {{< text bash >}}
-    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
-      --set istio_cni.enabled=true > $HOME/istio.yaml
+    $ kubectl create namespace istio-system
     {{< /text >}}
 
+1.  Install Istio CNI.
+
+    1.  Option 1.  Render and apply the Istio CNI Helm template.
```

Comment:
Do not use numbered lists for alternative options. The numbers tell readers that they should run one command after the other and that is not the case here. 

```suggestion
    - Option 1.  Render and apply the Istio CNI Helm template.
```